boards/BMS/Src/App/states/App_ChargeState.c

```diff
@@ -2,13 +2,22 @@
 #include "states/App_ChargeState.h"
 #include "states/App_FaultState.h"
 
-#include "App_SetPeriodicCanSignals.h"
 #include "App_SharedMacros.h"
 
+#define TOTAL_CHARGE_TIME_S (3600U) // 1 hr
```

Comment:
how was this time defined?

Reply:
arbitrary, we weren't too sure how to determine when charging is complete, but I think your suggestion of tracking the pack voltage is the most logical

Reply:
determining a completed charging should be done through future SOC firmware right?

Reply:
that or just checking if the max cell voltage has reached 4.2V